src/vue/backend/src/views/guided-setup/ListingsGeneral.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <GuidedSetupContentCard
+        :cardTitle="cardTitle"
+        :steps="guidedSetupSteps"
+        :relatedLinks="links"
+        @back-step="goBackStep"
+        @skip-step="goSkipStep"
+        @continue="goContinue">
+        <template v-slot:controls>
+            <ListingsGeneral/>
```

Comment:
This does not match the new form content flow implementation, and is not adjusting the state